cmake/SetupDoxygen.cmake

```diff
@@ -14,25 +14,64 @@ if (DOXYGEN_FOUND)
   # Configure file that contains doxygen groups
   configure_file(docs/GroupDefs.hpp ${SPECTRE_DOXYGEN_GROUPS})
 
-  add_custom_target(
-      doc
-      COMMAND ${DOXYGEN_EXECUTABLE} ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml 2>&1
-      DEPENDS
-      ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml
-      ${SPECTRE_DOXYGEN_GROUPS}
-      )
-
-  add_custom_target(
-    doc-check
-    # search for warnings and print out 6 lines after to make sure
-    # majority of warning is printed so the results are actually
-    # useful.
-    COMMAND ! ${DOXYGEN_EXECUTABLE} ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml 2>&1
-    | grep -A 6 'warning'
-    DEPENDS
-    ${PROJECT_BINARY_DIR}/docs/DoxyfileHtml
-    ${SPECTRE_DOXYGEN_GROUPS}
-    )
+  # We need Python for postprocessing the documentation
+  find_package(PythonInterp)
+  include(FindPythonModule)
+  find_python_module(bs4 FALSE)
+  find_python_module(pybtex FALSE)
```

Comment:
Add these to the page listing all the dependencies.

Do we want to get them on Travis?  Are they available on whatever generates our public documentation?

Reply:
I added them to the Docker container (simple `pip install`s). Is that enough to propagate to travis and public dox?

Reply:
Once the docker image is updated, yes. I can do that at some point soonish.